app/controllers/admin/works_controller.rb

```diff
@@ -192,11 +192,12 @@ def create_combined_audio_derivatives
   # recursive CTE so it'll be efficient-ish.
   def publish
     authorize! :publish, @work
-
     @work.class.transaction do
       @work.update!(published: true)
-      @work.all_descendent_members.find_each do |member|
-        member.update!(published: true)
+      unless params[:cascade] == "false"
+        @work.all_descendent_members.find_each do |member|
```

Comment:
I'm still wondering about leaving the "default", when no `params[:cascade]` is present at all -- to be cascading. Since cascading is the more 'dangerous' operation, it seems like maybe if no `params[:cascade]` is present at all -- or `params[:cascade]` is set to some nonsense like `adf `--  it should NOT cascade.  

I think `params[:cascade] == 'true'` is also a more usual pattern, that matches how we do boolean params everywhere else, instead of `params[:cascade] != 'false'`

Reply:
Yeah -- that's how I wrote it initially! I couldn't make up my mind... so thanks for weighing in.